### DISCUSSION TITLE
Can boogie output the runtime?

### DISCUSSION THREAD

**Discussion by Luweicai** [boogie-org/boogie#474] [Q&A] [answered]
It seems that there is no such option.

**Comment by shazqadeer** [comment, marked as answer]
---- Debugging and general tracing options ---------------------------------

  /trace        blurt out various debug trace information
  /traceTimes   output timing information at certain points in the pipeline
  /tracePOs     output information about the number of proof obligations
                (also included in the /trace output)
  /break        launch and break into debugger
